README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Coming Soon
 
-We are refactoring our [internal LLM training system](https://arxiv.org/abs/2402.15627) components to meet open source standard. The tentative timeline is as follows:
```

Comment:
@lichen225 can should no update README.md in github

Reply:
Have it restored to the previous version